src/main/java/org/folio/circulation/domain/UpdateItem.java

```diff
@@ -48,13 +47,6 @@ public UpdateItem(Clients clients) {
       Request request = requestQueue.getHighestPriorityFulfillableRequest();
 
       String pickupServicePointIdString = request.getPickupServicePointId();
-      if (pickupServicePointIdString == null) {
```

Comment:
Whilst it should be more unlikely after this change, the storage modules does not implement conditional validation to protect against this, so it could still happen.

Has this been protected against elsewhere, or can we reinstate this so that there is still an interpretable error under these circumstances?

Unless it has been discussed that we want to allow check in under these circumstances, it just wouldn't fulfil the request?

Reply:
The reason I ended up removing it was because I couldn't come up with a test strategy that could trigger the condition after updating the ServicePointPickupLocationValidator to prevent the condition from occurring.

I can reinstate the check, but I'm still not sure how to test for it.

Reply:
I've restored the last mile check in UpdateItem, but I'm still not sure how to trigger the condition from a test.

Reply:
One way to do it could be to remove the pickup service point from the request by going direct to storage.

In the broader sense, this is the kind of situation where a more unit like test could be useful.

Reply:
I've added a test for this (a modified version of the one that was removed)